autoload/pymode/breakpoint.vim

```diff
@@ -9,7 +9,7 @@ fun! pymode#breakpoint#init() "{{{
 
         PymodePython << EOF
 
-from imp import find_module
+from importlib import find_module
```

Comment:
This is not possible, as many people pointed out, because `importlib.find_module` does not exist.

For this file, you can use  `importlib.utils.find_spec`. You can use that to write the following:
```
from importlib.util import find_spec

for module in ('wdb', 'pudb', 'ipdb', 'pdb'):
    mod_ = find_spec(module)
    if mod_ is not None:
        vim.command('let g:pymode_breakpoint_cmd = "import %s; %s.set_trace()  # XXX BREAKPOINT"' % (module, module))
        break
```

To preserve python 2 / old python compatibility, you may want to write a wrapper around `imp.find_module` so it behaves a bit like find_spec:

```
def find_spec(name):
    try:
        from importlib.util import find_spec
        return find_spec(name)
    except ImportError:
        try:
            from imp import find_module
            find_module(name)
            return name
        except:
            return None
````

Reply:
would fix it, just had time! Sorry about that